lib/ret/hub.ex

```diff
@@ -279,14 +288,60 @@ defmodule Ret.Hub do
     end
   end
 
+  defp add_default_perms_to_changeset(changeset) do
+    default_perms = %{
+      spawn_and_move_media: true,
+      spawn_camera: true,
+      spawn_drawing: true,
+      pin_objects: true
+    }
+
+    changeset |> put_change(:permissions, default_perms |> hub_perms_to_int!)
+  end
+
+  @hub_perms %{
+    0x0000_0001 => :spawn_and_move_media,
```

Comment:
to reduce typo risk somewhat, I usually like writing these kinds of things like `1 << 0`, `1 << 1`, etc. minor tho